modules/test/playwright/tests/site-cms-site-initializer/vocabularies.spec.ts

```diff
@@ -0,0 +1,167 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {clickAndExpectToBeVisible} from '../../utils/clickAndExpectToBeVisible';
+import {getRandomInt} from '../../utils/getRandomInt';
+import getRandomString from '../../utils/getRandomString';
+import {waitForAlert} from '../../utils/waitForAlert';
+import {cmsPagesTest} from './fixtures/cmsPagesTest';
+
+const test = mergeTests(
+	cmsPagesTest,
+	featureFlagsTest({
+		'LPD-17564': {enabled: true},
+	}),
+	loginTest()
+);
+
+test(
+	'Assert can delete vocabulary from dropdown actions ',
+	{tag: '@LPD-32750LPD-32750LPD-32750LPD-32750LPD-32750LPD-32750'},
+	async ({editVocabularyPage, page, vocabulariesPage}) => {
+		editVocabularyPage.goto();
+
+		const name = `Vocabulary${getRandomInt()}`;
+
+		await editVocabularyPage.changeGeneralInfo({
+			description: getRandomString(),
+			name,
+		});
+
+		await editVocabularyPage.saveButton.click();
+
+		await waitForAlert(page, 'successfully', {timeout: 5000});
+
+		await vocabulariesPage.execItemAction({
+			action: 'Delete',
+			filter: name,
+		});
+
+		await expect(
+			page.getByRole('heading', {name: `Delete "${name}"`})
+		).toBeVisible();
+
+		const deleteButton = page.getByRole('button', {name: 'Delete'});
```

Comment:
@noornajj can you use the provided helper tasks ClickAndExpectToBeVisible and waitForAlert